### ISSUE TITLE
Not executable - No information from the manufacturer - Malicious code likely

### ISSUE THREAD

**Issue by ghost** [verza22/glarium#31]
The functionality of "Glarium" is not confirmed anywhere. Dropped PHP code and you should now find out how to get it to work. Even a pro can't get this script to run with the given installation instructions. There is simply a lack of important information that is intentionally withheld. The prerequisite for this script node does not make it better, rather very questionable. So don't worry, you will never get this script to work and you don't want it either. This script achieves something completely different. Who hides which malicious code behind it, so keep your hands off.

**Comment by Indianos** [comment]
What exactly is there malicious?

**Comment by ghost** [comment]
> What exactly is there malicious?

Most scripts created with Nodjs have malicious code. How are you supposed to find out in which script something really exists if you are not familiar with Node js yourself. All you can do is look out for other signs. What I hereby note. No matter what question you ask, nothing comes from the developer. Either the script has been abandoned or he is no longer present so he can no longer react. Or it could also be an incomplete script which does not work at all in this template. However, it could also contain malicious code. Everything is possible, so keep your distance and stay away. I basically do this for questionable scripts that don't work and have to do with Node JS. Many spread extremely dangerous malicious code with Node JS, that's a real hype.

**Comment by Indianos** [comment]
I have got it up and running. The only danger there is out-of-date packages. Do not write absurd claims.

**Comment by luidyreis** [comment]
> Eu tenho-o instalado e funcionando. O único perigo são os pacotes desatualizados. Não escreva afirmações absurdas.

Hello, could you help me with the installation?

**Comment by ghost** [comment]
There is no installation there. The script doesn't work. You need very high level of skill to run this garbage and when you have that you will find that the script is not playable and therefore uninteresting and above all does some things that are very strange. Many currently use Python to run malware. That's why I would basically stay away from scripts that only run with Python. I've had very bad experiences there. Stay away from this garbage and especially from Phyton environments. In general, stay away from all these things, they require a lot of resources on your server. All the illegally cheaply copied scripts like Travian are very bad and really no fun. I've been installing shit like this for my friends on servers for years. There was always something going on on the servers or the scripts were extremely slowing down the servers. Everything just garbage. None of this is usable. Believe me. I mean well. Don't let anyone tell you otherwise and use your time more wisely. There are also scripts that run and you don't need anything more than PHP (and of course a server) but even those are boring. The time is just over. Today it's all about the money. Almost all of them no longer run with PHP 8 , so that's uninteresting anyway.

**Comment by ghost** [comment]
> I have got it up and running. The only danger there is out-of-date packages. Do not write absurd claims.

Exactly, that's what I mean! Look at what "Indianos" wrote. And that's the kind of help you get. It's always the same guys who just babble on instead of writing here or now writing here how he did it. The developer writes nothing, the users write nothing. Everyone seems to install it wonderfully, showing you screenshots or test pages from completely different installations. Has nothing to do with this script here. But nobody can write you instructions, always the same, everywhere. Believe me, it's exactly the same dilema with Travian Kingdoms. The script is running, but everyone who logs on to this script on your server will get spawn emails and other shit, and then they'll also notice that the game isn't finished and you basically can't play it at all. Conclusion: Leave your stupid sayings and write instructions for your scripts, everything else is pointless and just kidding. As long as this is not clear here. Can you expect malware from this script?

**Comment by Indianos** [comment]
@AndyLeecher Please provide which part of code is even potentially malicious? Let's review it as a community. If you have nothing to pass on, don't insult others.
Since I haven't been on the PC I have got it up and running I also haven't provided any more details for setup but will do once will boot up that PC. All I did was set up docker and besides already mentioned services (PHP & ubuntu) I searched for compatible NodeJS version. It was 13 or around there cannot remember precisely.

**Comment by ghost** [comment]
> @AndyLeecher Please provide which part of code is even potentially malicious? Let's review it as a community. If you have nothing to pass on, don't insult others. Since I haven't been on the PC I have got it up and running I also haven't provided any more details for setup but will do once will boot up that PC. All I did was set up docker and besides already mentioned services (PHP & ubuntu) I searched for compatible NodeJS version. It was 13 or around there cannot remember precisely.

Hello Indianos, you really ran the script respect. I hope you used a local server for this. Of course, if you hand things over to an automated installer script and then don't remember what was done to make the script run, I can imagine that this is in the interest of the developer. I don't use something like that, it doesn't work at all. I always want to know what is happening on my server. So far, I could also manually install any complicated script with far more requirements. This didn't work here. Everything is designed so you can't do that and that's what I find strange. I'm not saying that this script contains malware, but whoever installs it cannot rule it out 100%. Please read my other comments on this. I never write that there is malware in this script, I always write "It could be". Node JS does the rest. Don't get me wrong, I appreciate the work behind it, it takes a lot of time. Mainly because this is also Ilegall. I have no idea what happens when the real developer of this game finds out what is being deliberately recreated here, no matter what the title is renamed to. It is an illegal replica, that alone is enough to avoid something like that. So leave it. Nevertheless, I respect you for installing something like this on your server. really hard.

**Comment by verza22** [comment]
Apologies for not commenting earlier; I am updating the code and setting up Docker so it can be installed easily.

**Comment by piposeimandi** [comment]
Buenas , creo que esto no es in issue real , al menos no tiene ninguna intención de que se arregle o mejore algo , deberías cerrarlo @verza22
